.stylelintrc.yml

```diff
@@ -0,0 +1,21 @@
+extends: stylelint-config-standard
```

Comment:
I think this should maybe added to the local .stylelintrc file on your computer and maybe not to the repo

Reply:
Removed. I added this accidentally with the my .log